scripts/script.js

```diff
@@ -0,0 +1,11 @@
+
+function showInfo(name,breed,fee) {
+  result = (name + ' is a ' + breed + ' and can be adopted for ' + fee + '.');
```

Comment:
Make sure you always declare a variable with 'let' 'const' etc... its generally bad practice to declare a variable in javascript without a keyword because it automatically becomes global and can lead to bugs.